ga4gh/datamodel/rna_quantification.py

```diff
@@ -0,0 +1,390 @@
+        """
+        protocolElement = protocol.RnaQuantification()
+        protocolElement.annotationIds = self._annotationIds
+        protocolElement.description = self._description
+        protocolElement.id = self.getId()
+        protocolElement.name = self._name
+        protocolElement.readGroupId = self._readGroupId
+        return protocolElement
+
+    def addRnaQuantMetadata(self, fields):
+        """
+        data elements are:
+        Id, annotations, description, name, readGroupId
+        where annotations is a comma separated list
+        """
+        self._annotationIds = fields["annotation_ids"].split(',')
+        self._description = fields["description"]
+        self._name = fields["name"]
+        self._readGroupId = fields["read_group_id"]
+
+
+class RNASeqResult(AbstractRNAQuantification):
+    """
+    Class representing a single RnaQuantification in the GA4GH data model.
+    """
+
+    def __init__(self, parentContainer, localId, rnaQuantDataPath,
+                 dataRepository):
+        super(RNASeqResult, self).__init__(parentContainer, localId)
+        self._dbFilePath = rnaQuantDataPath  # the full path of the db file
+        self._dataRepository = dataRepository
+        self._db = SqliteRNABackend(self._dbFilePath)
+        self.getRnaQuantMetadata()
+
+    def getRnaQuantMetadata(self):
+        """
+        input is tab file with no header.  Columns are:
+        Id, annotations, description, name, readGroupId
+        where annotation is a comma separated list
+        """
+        rnaQuantId = self.getLocalId()
+        with self._db as dataSource:
+            rnaQuantReturned = dataSource.getRnaQuantificationById(
+                rnaQuantId)
+        if rnaQuantReturned is not None:
+            self.addRnaQuantMetadata(rnaQuantReturned)
+        else:
+            raise exceptions.RnaQuantificationNotFoundException(rnaQuantId)
+
+    def getExpressionLevels(self, rnaQuantID, pageToken=0, pageSize=None,
+                            expressionId=None, featureGroupId=None,
+                            threshold=0.0):
+        """
+        Returns the list of ExpressionLevels in this RNA Quantification.
+        """
+        if expressionId is not None:
+            rawId = datamodel.ExpressionLevelCompoundId.deobfuscate(
+                expressionId)
+            expressionId = string.split(rawId, ":")[-1]
+        with self._db as dataSource:
+            expressionsReturned = dataSource.searchExpressionLevelsInDb(
+                rnaQuantID, pageToken=pageToken,
+                pageSize=pageSize, expressionId=expressionId,
+                featureGroupId=featureGroupId, threshold=threshold)
+        return [ExpressionLevel(self, expressionEntry) for
+                expressionEntry in expressionsReturned]
+
+    def getExpressionLevel(self, compoundId):
+        rawId = datamodel.ExpressionLevelCompoundId.deobfuscate(compoundId)
+        localExpressionId = string.split(rawId, ":")[-1]
```

Comment:
```python
parsedId = datamodel.ExpressionLevelCompoundId.parse(compoundId)
localExpressionId = parsedId.expression  # or whatever the name is.
```
We shouldn't be calling deobfuscate and parsing ID strings here, it should be done by the CompoundId machinery.